src/Resources/config/services.yml

```diff
@@ -80,6 +80,56 @@ services:
         calls:
             - ["setContainer", ["@service_container"]]
 
+    contao.menu.matcher:
+        class: Knp\Menu\Matcher\Matcher
+        public: false
+
+    contao.menu.picker_menu_builder:
+        class: Contao\CoreBundle\Menu\PickerMenuBuilder
```

Comment:
Is’t `menu` here duplicated? Would `contao.menu.picker_builder` and `Menu\PickerBuilder` be better?